lib/json-schema/util/uri.rb

```diff
@@ -1,109 +1,131 @@
+# frozen_string_literal: true
+
 require 'addressable/uri'
 
 module JSON
   module Util
-    module URI
+    class URI < Addressable::URI
```

Comment:
I'm wondering if this is a breaking change or not and I'm a bit unsure. What is your opinion @bolshakov ?

Reply:
To be honest, I haven't thought of this helper as part of the public API. I would consider it rather private, to be used internally within the gem. The only change my PR introduces is that you cannot include URI as a module anymore since it's a class now. However, this feature is not documented in the README, so I wouldn't consider it part of the public API.

A good idea might be to add a comment explicitly stating this:

```suggestion
    # @api private 
    class URI < Addressable::URI
```

Reply:
I also don't think this is really a public API, but it might be used as one. I also want to do a major release to drop support for old rubies. So maybe we don't release this one as a Bugfix but with the next major release?

Reply:
Your plan sounds good to me. Would you prefer adding `@api private` tag?

Reply:
I think that makes sense. Can you please add it?